vault/dendron.topic.links.md

```diff
@@ -19,6 +19,7 @@ Dendron supports multiple types of links and formats.
 ## Wiki Links
 Wiki links support **autocomplete**. To initiate:
 - create wikilink brackets and start typing
+- link to specific sections of notes using the `[[wiki link#header]]` syntax
```

Comment:
currently relative wiki links will now work in preview or when exported as a pod but they will work in publishing. can we add those caveats?

Reply:
Yep, will do.